packages/lutin/lutin.2.71.10/opam

```diff
@@ -0,0 +1,47 @@
+opam-version: "2.0"
+synopsis: "Lutin: modeling stochastic reactive systems"
+description: """
+Lutin is a language to model stochastic reactive systems. It has been
+initially designed to model environments and perform automated testing
+of reactive systems with Lurette.
+"""
+maintainer: "erwan.jahier@univ-grenoble-alpes.fr"
+authors: [
+  "Erwan Jahier" "Pascal Raymond" "Yvan Roux"
+]
+license: "CECILL-2.1"
+tags: ["clib:stdc" "clib:camlidl"]
+homepage: "http://www-verimag.imag.fr/Lutin.html"
+dev-repo: "git+https://gricad-gitlab.univ-grenoble-alpes.fr/verimag/synchrone/lutin"
+bug-reports: "https://gricad-gitlab.univ-grenoble-alpes.fr/verimag/synchrone/lutin/issues" 
+depends: [
+  "ocaml"
+  "base-bigarray"
+  "conf-gmp"
+  "conf-mpfr"
+  "conf-m4" {build}
+  "conf-perl" {build}
+  "base-unix" {build}
+  "num"
+  "camlidl"
+  "graphics"
+  "extlib" {build} | "extlib-compat" {build}
+  "lustre-v6"
+  "lutils" {>= "1.49"}
+  "mlgmpidl"
+  "dune" {>= "2.0"}
+  "ocamlfind"
+  "rdbg" {>= "1.196.9"}
+]
+build: ["dune" "build" "@install" "-j" jobs]
+install: [ make "install" ]
```

Comment:
Why has the `-p` argument been removed? make install shouldn't be required either.

Reply:
I've removed the -p argument to fix the build but you fix in 0a48d3a8e7c24a127a33650db54902fc6b2c8cba is better indeed (I tough I've tried something like that!), thanks.

Without the install, the ci is happy but nothing is actually installed